pkg/pods/manifest.go

```diff
@@ -41,6 +41,14 @@ type Manifest struct {
 	signature []byte
 }
 
+type JSONManifest struct {
```

Comment:
You don't need to add a new type to specify the JSON encoding for these fields. Just reuse `Manifest`. Go struct tags can have multiple key-value pairs if you separate them with spaces [[ref]](http://golang.org/pkg/reflect/#StructTag).

```go
Id string `yaml:"id" json:"id"`
```